### PR TITLE
Add Francis' project portfolio and image

### PR DESCRIPTION
**Pull request by fustilio**
Add Francis' project portfolio and image

### CODE REVIEW

File: docs/AboutUs.adoc [hunk 1]
```diff
@@ -55,6 +55,12 @@ Responsibilities: UI
 
 '''
 
+=== Lee Jun Yao Francis
+image::francis.jpg[width="150", align="left"]
+{empty}[https://github.com/fustilio[github]] [<<francislee#, portfolio>>]
+
+Role: Code Monkey
+
```
Comment (Kennard123661) [inline docs/AboutUs.adoc]: Missing Responsibility: TBA here

Comment (Kennard123661) [reply docs/AboutUs.adoc]: @fustilio

**Comment by fustilio** [comment]
Add deleteTag in Model component

**Comment by Kennard123661** [comment]
Please split your portfolio pull request and your enhancement pull request @fustilio

**Comment by fustilio** [comment]
Finally things looks good again.

**Pull request merged**